src/test/java/com/VetApp/PurrgentCare/FakeDataGenerator.java

```diff
@@ -106,33 +103,77 @@ public List<Person> generatePersonList(Integer countOfFakePerson) {
         return fakePeopleList;
     }
 
+    public List<Person> generateDefaultPersonList() {
+        final var numberFakePersons = new Random().nextInt(1, 5);
+        return generatePersonList(numberFakePersons);
+    }
+
     /////
     //Account Types
     /////
-    public Account generateAccount(Integer fakeAccountId, Boolean fakeActive, Date fakeDateCreated, List<Pet> fakePets, List<Person> fakeAccountHolders) {
+    public Account generateFakeAccount(Integer fakeAccountId, Boolean fakeActive, Date fakeDateCreated, List<Pet> fakePets, List<Person> fakeAccountHolders) {
         return Account.builder().id(fakeAccountId).active(fakeActive).dateCreated(fakeDateCreated).pets(fakePets).accountHolders(makeListMutable(fakeAccountHolders)).build();
     }
 
+    public Account generateDefaultAccount() {
+        return Account.builder()
+                .id(generateRandomInteger())
+                .active(generateRandomBoolean())
+                .dateCreated(generateRandomDate())
+                .pets(generateDefaultPetList())
+                .accountHolders(generateDefaultPersonList())
+                .build();
+    }
+
     public List<Account> generateAccountList(Integer countOfFakeAccounts) {
         List<Account> fakeAccountList = new ArrayList<>(List.of());
         var i = 1;
         while (i <= countOfFakeAccounts) {
-            var fakeAccount = generateAccount(generateRandomInteger(), generateRandomBoolean(), generateRandomDate(), generateDefaultPetList(), generatePersonList(generateRandomInteger()));
+            var fakeAccount = generateFakeAccount(generateRandomInteger(), generateRandomBoolean(), generateRandomDate(), generateDefaultPetList(), generatePersonList(generateRandomInteger()));
             fakeAccountList.add(fakeAccount);
             i++;
         }
         return fakeAccountList;
     }
 
-    public AccountResponse generateAccountResponse(Boolean fakeActive, Date fakeDateCreated, List<Pet> fakePets, List<Person> fakeAccountHolders) {
+    public List<Account> generateDefaultAccountList() {
+        final var numberFakeAccounts = new Random().nextInt(1, 5);
+        return generateAccountList(numberFakeAccounts);
+    }
+
+    public AccountRequest generateFakeAccountRequest() {
+        return AccountRequest.builder().active(generateRandomBoolean()).dateCreated(generateRandomDate()).build();
+
+    }
+
+    public AccountResponse generateFakeAccountResponse(Integer fakeId, Boolean fakeActive, Date fakeDateCreated, List<Pet> fakePets, List<Person> fakeAccountHolders) {
+        final var accountResponse = new AccountResponse();
+        accountResponse.setId(fakeId);
+        accountResponse.setActive(fakeActive);
+        accountResponse.setDateCreated(fakeDateCreated);
+        accountResponse.setPets(fakePets);
+        accountResponse.setAccountHolders(fakeAccountHolders);
+        return accountResponse;
```

Comment:
Can we convert these to use the Builder methods?

Reply:
Converted